backend/server/constants.go

```diff
@@ -16,9 +18,9 @@ const FAQ_URL = "api/v1/faq"
 const RESOURCES_URL = "api/v1/resources"
 
 // JWT used for testing
-const AUTH_TOKEN = "Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhZG1pbiI6dHJ1ZSwiZXhwIjoxNjA2Mjc5MDg2LCJ6SUQiOiJ6NTEyMzQ1NiJ9.HElUm-Qmj9asFcemvD-8Na4dIXNEZpft3BzANxUS6ZU"
+var AUTH_TOKEN = "Bearer " + os.Getenv("TESTING_JWT")
 
-var JWT_SECRET = []byte("temp_secret_until_proper_secrets_are_implemented")
+var JWT_SECRET = []byte(os.Getenv("JWT_SECRET"))
```

Comment:
Have you tried debugging this to see whether this actually retrieves the env variable?

Reply:
It doesn't - it's returning empty strings, and I'm figuring out why.

Reply:
Looking at the documentation for it. It looks like you have to pass it into through a go command on the command line or set it when the server has launched. You might have to change the dockerfile to inject these before the server goes up.